src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.html

```diff
@@ -1,35 +1,25 @@
-<p-dialog visible="true" [draggable]="false" [modal]="true" [dismissableMask]="true" (onHide)="hideDialog()" header="{{ dotMessageService.get('editpage.layout.theme.header') }}">
+<dot-dialog show="true" [ok]="applyDialogAction" [cancel]="closeDialogAction" (close)="hideDialog()" header="{{ dotMessageService.get('editpage.layout.theme.header') }}">
     <dot-site-selector #siteSelector (change)="siteChange($event)" [archive]="false"></dot-site-selector>
     <div class="dot-theme-search-box">
-        <dot-icon class="searchable-dropdown__search-icon" name="search" tiny="true"></dot-icon>
-        <input #searchInput pInputText placeholder="{{ dotMessageService.get('editpage.layout.theme.search') }}" type="text" class="searchable-dropdown__search-field"/>
+        <dot-icon class="searchable-dropdown__search-icon" name="search"></dot-icon>
+        <input #searchInput pInputText placeholder="{{ dotMessageService.get('editpage.layout.theme.search') }}" type="text" class="searchable-dropdown__search-field"
+        />
     </div>
     <div class="dot-themes-container">
-        <p-dataGrid #dataGrid
-            [value]="themes | async"
-            [paginator]="true"
-            [rows]="paginatorService?.paginationPerPage"
-            [totalRecords]="paginatorService?.totalRecords"
-            [lazy]="true"
-            (onLazyLoad)="paginate($event)"
-            [emptyMessage]="dotMessageService.get('editpage.layout.theme.no.records.found')"
+        <p-dataGrid #dataGrid [value]="themes | async" [paginator]="true" [rows]="paginatorService?.paginationPerPage" [totalRecords]="paginatorService?.totalRecords"
```

Comment:
When we have a element with a bunch of attr is easier to read when each attr is in one line.